### PR TITLE
feat: early access pricing

### PR DESCRIPTION
**Pull request by vinckr**
Since https://github.com/ory/docs/pull/507 is merged and live now, we should update pricing also. 
This is the first attempt with only the developer and enterprise tier visible, we will add the rest soon!

### CODE REVIEW

**Review by jfcurran — APPROVED** [review]

File: src/page-content/pricing-tiers/pricing-saas.tsx [hunk 1]
```diff
@@ -11,10 +11,11 @@ export const pricingSaas: Pick<PricingProps, 'saas'> = {
   saas: [
     {
       theme: 'dark',
-      title: <>Early Access</>,
-      description: 'Request early access through the Ory Insider program',
-      price: '$99',
-      priceDescription: 'monthly per project',
+      title: <>Developer</>,
+      description: 'Early Access for developers',
```
Comment (aeneasr) [inline src/page-content/pricing-tiers/pricing-saas.tsx:15]:
```suggestion
      title: <>Developer</>,
      description: 'Free for developers, no credit card needed',
```

File: src/page-content/pricing-tiers/pricing-saas.tsx [hunk 2]
```diff
@@ -11,10 +11,11 @@ export const pricingSaas: Pick<PricingProps, 'saas'> = {
...
+      price: 'Free',
+      priceDescription: 'for one project',
```
Comment (aeneasr) [inline src/page-content/pricing-tiers/pricing-saas.tsx:17]:
```suggestion
      priceDescription: 'forever',
```

File: src/components/layouts/pricing/pricing.tsx [hunk 3]
```diff
@@ -58,9 +58,8 @@ const Pricing = ({ id, title, description, saas, opensource }: PropTypes) => {
           <Grid lg={8} md={8} sm={12} xs={12}>
             <Container justify={'center'} className={'text-is-centered'}>
               <p className={cn('font-p-small')}>
-                <span className={'is-themed-primary'}>* </span>Fair use of
-                database storage for identities, flows and sessions. Upgrades
-                are possible.
+                <span className={'is-themed-primary'}>* </span>Upgrades
```
Comment (aeneasr) [inline src/components/layouts/pricing/pricing.tsx:61]: I would keep the fair use section - because "production plan" is even more vague :D

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: src/page-content/pricing-tiers/pricing-saas.tsx [hunk 4]
```diff
@@ -23,28 +24,40 @@ export const pricingSaas: Pick<PricingProps, 'saas'> = {
...
+          title: 'Bring your own UI'
+        },        
+        {
+          icon: CheckCircle,
+          title: 'Custom Identity Models'
         },
         {
           icon: CheckCircle,
-          title: 'Ory CLI'
+          title: '500 Identities*'
```
Comment (aeneasr) [inline src/page-content/pricing-tiers/pricing-saas.tsx:51]: This looks like we do pricing per identity

File: src/components/layouts/pricing/pricing.tsx [hunk 5]
```diff
@@ -58,9 +58,8 @@ const Pricing = ({ id, title, description, saas, opensource }: PropTypes) => {
...
+                to production plans are possible, please contact us.
```
Comment (aeneasr) [inline src/components/layouts/pricing/pricing.tsx:62]: Developer projects are limited to X identities. All other plans have no identity limits. This is very important, but it is not clear from the current proposal.

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by vinckr** [comment]
How do you like this @aeneasr 
It is not super prominent, but we can fix that in the bigger overhaul?

**Review by BlueAcidFrog — APPROVED** [review]

**Pull request merged**